### ISSUE TITLE
MESH term mapping

### ISSUE THREAD

**Issue by Crispae** [Crispae/pbpko#2]
As MESH is considered as thesaurus, and OBO mention importing terms only from ontology. So for terms available in MESH, we will use standard [`database_cross_reference`](http://purl.obolibrary.org/obo/database_cross_reference) annotation.

For exmaple in uberon-phenotype ontology, mesh terms were linked with cross refrence annotation.  
https://github.com/obophenotype/uberon

This issue is  open to discuss about how MESH terms should be referenced in OBO ontologies.

**Comment by reality** [comment]
Yes, normally would just be dbCrossRef annotations. if we want to be fancy we can also define something in https://mapping-commons.github.io/sssom/
